vmm/src/interrupt.rs

```diff
@@ -115,14 +115,6 @@ impl MsiInterruptGroup<IrqRoutingEntry> {
     }
 }
 
-pub trait RoutingEntryExt {
```

Comment:
I think keeping the trait is more idiomatic for Rust, and provides an abstraction layer. I don't think we should drop it.